ansible_base/authentication/authenticator_plugins/utils.py

```diff
@@ -53,7 +53,6 @@ def get_authenticator_urls(authenticator_type: str) -> list:
     return []
 
 
-def generate_authenticator_slug(type: str, name: str, suffix: Optional[str] = None) -> str:
-    clean_type = f"{type.replace('.', ' ')}"
-    slug_template = f"{clean_type}__{name}" if not suffix else f"{clean_type}__{name}__{suffix}"
+def generate_authenticator_slug(name: str) -> str:
+    slug_template = f"{name}__{uuid.uuid4()}"
```

Comment:
We really just need a unique identifier so that we can figure out which authenticator should get the response from an SSO attempt. One of the problems we have is a bit of a chicken and egg where some providers want the information to make the client secret but this information can be pre-determined easily so you have to save the client (which needs the client secret). Instead of having a uuid in here, which is going to be a bit more cringy for pre-determining the endpoint names, what if we just made a unique string field with a default of a uuid? The user could decide on a random phrase like `ardvark-sandwich` so that they could pre-configure the SSO authenticator. It could be optional in the UI but if we get nothing we either makeup some words or insert a UUID in there. Thoughts?

Reply:
I love aardvark sandwiches.

That seems reasonable though to allow the (extension portion?) to be specified optionally.  As long as it validates there aren't any collisions then it sounds good and should make the dev env update easier..

Reply:
I'm okay with this. so it would be provide a slug or have one generated and just be a uuid?